backend/de.metas.business/src/main/java/de/metas/quantity/Quantity.java

```diff
@@ -61,6 +61,16 @@ public static Quantity of(@NonNull final BigDecimal qty, @NonNull final I_C_UOM
 		return new Quantity(qty, uomRecord);
 	}
 
+	@Nullable
+	public static Quantity ofOrNull(@Nullable final BigDecimal qty, @Nullable final I_C_UOM uom)
```

Comment:
usually we call those methods `ofNullable`

Reply:
I've seen both variants used in an inconsistent manner. Will prefer `ofNullable` from now on.